apps/web/src/config/index.ts

```diff
@@ -14,6 +14,7 @@ import {
   MAIL_SERVER_DOMAIN,
   LAUNCH_DARKLY_CLIENT_SIDE_ID,
   FEATURE_FLAGS,
+  HUBSPOT_PORTAL_ID,
```

Comment:
I don't think we should (or need to) import this from shared-web. We can fully encapsulate the HS form constants in shared-web.